src/coreComponents/physicsSolvers/solidMechanics/EFEMKernelsHelper.hpp

```diff
@@ -0,0 +1,155 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+
+/**
+ * @file EFEMKernelsHelper.hpp
+ */
+
+#ifndef GEOSX_PHYSICSSOLVERS_SOLIDMECHANICS_EFEMKERNELSHELPER_HPP_
+#define GEOSX_PHYSICSSOLVERS_SOLIDMECHANICS_EFEMKERNELSHELPER_HPP_
+
+
+#include "common/DataTypes.hpp"
+#include "common/DataLayouts.hpp"
+#include "common/GeosxMacros.hpp"
+#include "common/TimingMacros.hpp"
+#include "LvArray/src/output.hpp"
+#include "LvArray/src/tensorOps.hpp"
+
+namespace geosx
+{
+
+namespace EFEMKernelsHelper
+{
+
+template< int NUM_NODES >
+void ComputeHeavisideFunction( integer (& heaviside)[NUM_NODES],
```

Comment:
perhaps this should go in computational geometry or somewhere accessible to others.  it is fairly generic.

Reply:
or maybe in `FiniteElementBase` ?